arabic/dream/another-dream/It was a nice dream.md

```diff
@@ -0,0 +1 @@
+كنت أحاول الفرار من بعض المجرمين وبينما أركض تسائلت لماذا لا أواجه هؤلاء الأوغاد فاستدرت وواجهتهم حتى قضيت على أربعة رجال فى نصف دقيقة قبل أن ينادينى أحدهم بأن أسرع لأن ميعاد المدرسة سيفوتنى ان لم أستيقظ :) [nice dream](../another dream/It was a nice dream.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.